hacenv.yml

```diff
@@ -3,18 +3,38 @@ dependencies:
     - python=3.8
     - pip
     - pip:
-        - torch==1.10.0+cu111.moreh23.6.0
-        - moreh-driver==23.6.0
+        - torch==1.13.1+cu116.moreh24.3.0
+        - torchaudio==0.13.1
+        - torchvision==0.14.1
+        - moreh-driver==24.3.0
         - transformers==4.29.2
-        - datasets
+        - datasets>=2.14.6
         - protobuf
         - evaluate
-        - sentencepiece != 0.1.92
+        - sentencepiece==0.1.99
         - accelerate >= 0.12.0
         - mlflow
+        - accelerate==0.21.0
+        - jiwer
+        - librosa
+        - matplotlib
+        - nltk==3.8.1
+        - numpy==1.24.4
```

Comment:
I'm curious, which task pinned numpy to this version?

Reply:
Actually, there is no specific task that requires fixing the numpy version; I accidentally kept it